Makefile

```diff
@@ -96,7 +96,6 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
 	pip-compile --upgrade -o requirements/production.txt requirements/production.in
 	# Let tox control the Django version for tests
-	grep -e "^django==" requirements/production.txt > requirements/django.txt
```

Comment:
Could we keep this and start using `django.txt` for the Django 2.2 tests in `tox.ini`?  As written, there's no guarantee tox will test with the exact version of Django being deployed to production.

Reply:
Updated `Makefile` and `tox.ini` to use `django.txt` file for testing `Django2.2`.